pytest/tests/sanity/epoch_switches.py

```diff
@@ -26,6 +26,7 @@
                 "nanos": 0
             },
             "state_sync_enabled": True,
+            "store.state_snapshot_enabled": True,
```

Comment:
I don't want to look inside pytest architecture, but it looks like we are doing lots of manual changes to config in the case when state sync is required. Should we make some generic function `enable_state_sync(node_config)` or add a parameter to `start_cluster(..., enable_state_sync=True)`?

Reply:
Good idea for a future cleanup of tests :+1: